tap_trello/streams.py

```diff
@@ -118,21 +118,18 @@ def get_records(self, format_values, params=None):
         if window_sub_end is not None:
             # Indicates we need to resume, next_window_start should be present
             next_window_start -= timedelta(milliseconds=1) # To make next start inclusive
+            LOGGER.info('Paginating with window_sub_end')
```

Comment:
This message should probably say something like 
> Resuming interrupted pagination for date window {} to {}

With the parameters `window_start` and `window_sub_end`.

Reply:
We'll probably remove this, it was a debugging line and it's sibling line in the `else` block was removed